tests/test-backend-ops.cpp

```diff
@@ -6268,7 +6254,7 @@ static std::vector<std::unique_ptr<test_case>> make_test_cases_eval() {
             for (int n_mats : {4, 8}) {
                 for (int n_used : {1, 2, 4}) {
                     for (bool b : {false, true}) {
-                        for (int n : {1, 4, 5, 32, 129}) {
```

Comment:
Don't change the legacy UT cases which impact other backends, otherwise they are wrong.